cmd/podman/containers/rename.go

```diff
@@ -0,0 +1,53 @@
+package containers
+
+import (
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	renameDescription = "The podman rename command allows you to rename an existing container"
+	renameCommand     = &cobra.Command{
+		Use:   "rename CONTAINER NAME",
+		Short: "Rename an existing container",
+		Long:  renameDescription,
+		RunE:  rename,
+		Args:  cobra.ExactArgs(2), // TODO we can do better here - check that #1 is container ID, #2 is a name
+		//ValidArgsFunction: common.AutocompleteContainers,
```

Comment:
I recommend using `common.AutocompletePortCommand` as `ValidArgsFunction`. This will complete only the first argument as container name or id.
`ExactArgs(2)` looks fine. I am not sure why you would want to check for an ID. The code and the example also seem to allow a name as first arg.